packages/auth/src/platform_browser/recaptcha/recaptcha_loader.ts

```diff
@@ -112,7 +117,7 @@ export class ReCaptchaLoaderImpl implements ReCaptchaLoader {
     // In cases (2) and (3), we _can't_ reload as it would break the recaptchas
     // that are already in the page
     return (
-      !!_window().grecaptcha &&
+      !!_window().grecaptcha?.render &&
```

Comment:
Sorry for the late comment - `!!_window().grecaptcha?.render` can be replaced with `librarySeparatelyLoaded`, so this becomes:

return (this.librarySeparatelyLoaded && (hl === this.hostLanguage || this.counter > 0 || this.librarySeparatelyLoaded)

Can we just rewrite this as:

`return (hl === this.hostLanguage || this.counter > 0 || this.librarySeparatelyLoaded)`?

Reply:
See PR description, these 2 expressions were previously redundant as well before this code change, so I thought about fixing it but I didn't know the original intent, and if one of them was originally intended to be a different condition that also needed to be checked. I wanted to leave it up to someone more familiar with the code.